streams.yml

```
@@ -88,7 +88,7 @@ rhel-7-golang:
 rhel8:
   # the most recent release at present. since we yum update this, it does not need to float.
   # it is important that we not build from unreleased builds and release them.
-  image: registry-proxy.engineering.redhat.com/rh-osbs/ubi8:8.6-655
+  image: registry-proxy.engineering.redhat.com/rh-osbs/rhel-els:8.6-744
```

I don't feel good about basing this on a base image that won't ship, until we hear either that there will be an 8.6-els that ships or that it's fine to use an unshipped one. i realize this kicks the outcome we would like (built from correct content sets) down the road a bit, but i'd like to play it safe and not publish unreleased base images.
```suggestion
  image: registry-proxy.engineering.redhat.com/rh-osbs/ubi8:8.6-754
```